second commit

Elvira-KEI opened the pull request.

Elvira-KEI committed:
second commit

nikolay977 commented on My_second_Project/src/main/java/org/example/constantText.java:
```diff
@@ -0,0 +1,20 @@
+package org.example;
+//"ответы" для выпадающего списка вопросов в разделе «Вопросы о важном»
+public class constantText {
```
Исправь наименование классов в соответствии с правилами нейминга. Классы — имя начинается с заглавной буквы, если в имени несколько слов. Каждое слово пишется с заглавной буквы слитно. Например: `Praktikum`, `BestProgram`.

nikolay977 commented on My_second_Project/src/test/java/WebTest.java:
```diff
@@ -0,0 +1,62 @@
+            this.constantText = constantText;
+            this.index = index;
+       }
+        @Parameterized.Parameters
```
В параметризованных тестах для аннотации @Parameterized.Parameters лучше использовать аргумент name: @Parameterized.Parameters(name = "Тестовые данные: {0} {1}"), где {0}, {1} - индексы параметров. Это повысит информативность проверки.

nikolay977 commented on My_second_Project/src/test/java/WebTest.java:
```diff
@@ -0,0 +1,62 @@
+        public WebTest(String constantText, int index) {
+            this.constantText = constantText;
+            this.index = index;
+       }
```
Используй сочетание клавиш Ctrl+Alt+L (Option+Command+L для Mac) для расстановки пробелов и табуляции в соответствии с настройками Code Style в Idea.

nikolay977 commented on My_second_Project/src/test/java/WebTestOrder.java:
```diff
@@ -0,0 +1,94 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
```
Code Style. Убери лишние пропуски строк. Для зрительного разделения кода достаточно использовать один пропуск строки. Между фигурными скобками пропуски не нужны. Пропуски строк должны быть однообразными.

Elvira-KEI committed:
My second commit

-rename class
-code formatting according to Code Style settings
-changing the @Parameterized.Parameters annotation using the name argument
